DHFPROD-5830: fix for revert entity type

brucean52 opened the pull request:
### Description

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

timur-isangulov reviewed (approved):
Works fine, tested using scenario in the ticket.

Sanjeevani19 reviewed (approved):
One e2e test fails but not related to this PR so approving.

akshaysonvane reviewed (changes requested):
I'm seeing some weird behavior on revert https://drive.google.com/file/d/1hA35mwK0KC-Jo2tIWE6j1NXNeZ04P15U/view

brucean52 commented:
> I'm seeing some weird behavior on revert https://drive.google.com/file/d/1hA35mwK0KC-Jo2tIWE6j1NXNeZ04P15U/view

Fixed this. Please review again

akshaysonvane reviewed (changes requested):
Revert seems fine now. Seeing this issue while saving an entity: https://drive.google.com/file/d/1jsHFjVvQJtzT4xWwpzHrtfgFq6dv8JIo/view

brucean52 commented:
> Revert seems fine now. Seeing this issue while saving an entity: https://drive.google.com/file/d/1jsHFjVvQJtzT4xWwpzHrtfgFq6dv8JIo/view

Added the fix for saving an entity.

Sanjeevani19 reviewed (changes requested):
The sort checkmark is still enabled even after saving those changes to database, Once page is refreshed the checkmark disappears.

brucean52 commented:
> The sort checkmark is still enabled even after saving those changes to database, Once page is refreshed the checkmark disappears.

Added the fix for Save All and updated e2e tests to cover the regression issues found.

akshaysonvane reviewed (changes requested):
Revert and save looks good. Seeing this issue while updating the description: https://drive.google.com/file/d/1XHvNNqx7d5dmd5duLpyZgbHgBjTcVbgx/view

brucean52 commented:
> Revert and save looks good. Seeing this issue while updating the description: https://drive.google.com/file/d/1XHvNNqx7d5dmd5duLpyZgbHgBjTcVbgx/view

Added fix for entity description editing and added e2e test to cover this case.

akshaysonvane commented on marklogic-data-hub-central/ui/src/components/modeling/entity-type-table/entity-type-table.tsx:
```diff
@@ -215,7 +222,7 @@ const EntityTypeTable: React.FC<Props> = (props) => {
         let entityName = parseText[0];
         let entityDescription = parseText[1];
-
+        console.log('entity desc', entityDescription)
```
Can we get rid of this?

akshaysonvane reviewed (approved):
This looks really good.
